### PR TITLE
Splash Screen

### PR DESCRIPTION
**Pull request by shreya024**
Added a Splash Screen with logo and loading.

### CODE REVIEW

**Comment by shreya024** [comment]
@NegiAkash890 Kindly review my PR.

**Comment by NegiAkash890** [comment]
> @NegiAkash890 Kindly review my PR.

Thanks shreya for the PR I will pull your changes and review it soon !

File: src/components/Splash Screen/SplashScreen.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        height="150rem"
+      />
+      <br />
+      <PropagateLoader size="20" />
+    </div>
+  );
+};
+
+export default Preloader;
+//<PropagateLoader size="20" />
```
Comment (NegiAkash890) [inline src/components/Splash Screen/SplashScreen.js:21]: Can you remove the commented line here.

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,40 @@
...
-import './App.css';
-
-function App() {
-  return (
-    <div className="App"> 
+import Footer from "./components/Footer";
+import Main from "./components/Main";
+import NavBar from "./components/NavBar";
+import React, { useState, useEffect } from "react";
+
```
Comment (NegiAkash890) [inline src/App.js:5]: Please remove empty line from here too.

Comment (shreya024) [reply src/App.js]: I have made the requested changes, @NegiAkash890 kindly check now.

**Comment by shreya024** [comment]
Thank you :)

**Pull request merged**